project/CompilerSettings.scala

```diff
@@ -26,6 +26,7 @@ object CompilerSettings {
       "-feature",
       "-language:_",
       "-unchecked",
+      "-Xfatal-warnings",
```

Comment:
missing -unpacked

Reply:
What is -unpacked? I can't find such an option in scalac.